From ff784f4803ab33f5e3389e40d038d52d1e211843 Mon Sep 17 00:00:00 2001
From: Yuan Chao <yuanc.fnst@cn.fujitsu.com>
Date: Wed, 28 Aug 2019 16:12:27 +0900
Subject: [PATCH] [PATCH] setup.py: remove the setup_requires for
 setuptools-scm

The setup_requires argument forces the download of the egg file for setuptools-scm
during the do_compile phase.  This download is incompatible with the typical fetch
and mirror structure.  The only usage of scm is the generation of the _version.py
file and in the release tarball it is already correctly created

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Derek Straka <derek@asterius.io>

Signed-off-by: Yuan Chao <yuanc.fnst@cn.fujitsu.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index adbafb5..75fdd09 100644
--- a/setup.py
+++ b/setup.py
@@ -19,7 +19,7 @@ INSTALL_REQUIRES = [
 def main():
     setup(
         use_scm_version={"write_to": "src/_pytest/_version.py"},
-        setup_requires=["setuptools-scm", "setuptools>=40.0"],
+        setup_requires=["setuptools>=40.0"],
         package_dir={"": "src"},
         extras_require={
             "testing": [
-- 
2.17.1

